processor/ratelimitprocessor/gubernator_test.go

```diff
@@ -146,52 +145,11 @@ func TestGubernatorRateLimiter_RateLimit(t *testing.T) {
 
 			respErr = errors.New("nope")
 			err = rateLimiter.RateLimit(context.Background(), 1)
-			assert.EqualError(t, err, "error executing gubernator rate limit request: rpc error: code = Unknown desc = nope")
+			assert.EqualError(t, err, errRateLimitInternalError.Error())
 		})
 	}
 }
 
-func TestGubernatorRateLimiter_RateLimit_MetadataKeys(t *testing.T) {
```

Comment:
Why did you delete this test?

Reply:
Oh gosh, this is my worst PR ever, this was definetly a mistake :sweat: I fixed it

Reply:
thanks :)